src/main/kotlin/racingcar/RacingCarGame.kt

```diff
@@ -0,0 +1,23 @@
+package racingcar
+
+class RacingCarGame(private val carCount: Int, private val trialCount: Int) {
+    private val carList = List(carCount) { Car() }
+    val gameResult
+        get() = _gameResult.toList()
+    private val _gameResult = mutableListOf<List<Int>>()
+
+    fun play() {
+        for (c in 0 until trialCount) {
+            proceed()
+
+            record()
+        }
```

Comment:
```suggestion
    fun play() = repeat(trialCount) {
        proceed()
        record()
    }
```

[repeat](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/repeat.html)를 사용하면 주어진 횟수 만큼 반복을 할 수 있습니다.😊